src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -288,6 +321,10 @@ private boolean checkForOtpRunnerCompletion(String url) {
             e.printStackTrace();
             return false;
         }
+        if (otpRunnerStatus.nonce == null || !otpRunnerStatus.nonce.equals(deployJob.getNonce())) {
+            LOG.warn("otp-runner status nonce does not match deployment nonce");
+            return false;
+        }
```

Comment:
This containing method's name (`checkForOtpRunnerCompletion`) seems inaccurate. Maybe it should change to `checkForOtpRunnerSuccess`? Also, could this message and/or comments be more descriptive? What are the conditions that would render this check to return false.